frame/transaction-payment/src/lib.rs

```diff
@@ -72,6 +78,12 @@ pub trait Trait: frame_system::Trait {
 
 	/// Update the multiplier of the next block, based on the previous block's weight.
 	type FeeMultiplierUpdate: Convert<Multiplier, Multiplier>;
+
+	/// Author rewarded for transaction inclusion.
+	///
+	/// Its reward for including transaction is used to compute transaction priority. Thus the
+	/// value returned in-between blocks must be sensible in regards of Author reward.
+	type Author: Author<Self::AccountId>;
```

Comment:
also this associated type can be error prone as usually in-between block default account is returned, and maybe in-between blocks default account can receive some funds for some other reason making the computation of priority wrong.